include/vrv/harm.h

```diff
@@ -58,6 +59,14 @@ class Harm : public ControlElement,
      */
     virtual void AddChild(Object *object);
 
+    /**
+     * Transposition related. The int tracks where we have iterated through the string.
+     */
+    TransPitch GetRootPitch(unsigned int &pos);
```

Comment:
For this method and `GetBassPitch` it would be clearer to pass TransPitch as parameter (`const TransPitch& pitch`) and to return a boolean indicating success and failure.